src/components/Sourvey/Sourvey.tsx

```diff
@@ -17,6 +17,7 @@ export const Sourvey: React.FC = () => {
     updateAnswers(questionKeys)
```

Comment:
I hadn't even realized that we had misspelled survey. I'll take care of it in my branch.

Reply:
I think there was a confusion between survey and journey lol, at least for me